Sources/VGSCheckoutSDK/Core/Checkout/CheckoutDelegate/VGSCheckoutDelegate.swift

```diff
@@ -10,6 +10,18 @@ public protocol VGSCheckoutDelegate: AnyObject {
 	/// - Parameter requestResult: `VGSCheckoutRequestResult` object, holds result of checkout flow.
 	func checkoutDidFinish(with requestResult: VGSCheckoutRequestResult)
 
-	/// Tells the delegate that user cancelled checkout flow (closed checkout screen.)
+	/// Tells the delegate that user cancelled checkout flow (closed checkout screen).
 	func checkoutDidCancel()
+
+	/// Tells the delegate that save card succeeded and fin instrument created.
+	/// - Parameter data: `Data?` object, holds response data with created financial instrument for saved card.
+	/// - Parameter response: `URLResponse?` object, holds URL response object with created financial instrument for saved card.
+	func saveCardDidSuccess(with data: Data?, response: URLResponse?)
+}
+
+/// Defeault protocol implementation.
+public extension VGSCheckoutDelegate {
```

Comment:
should we comment this code for now?

Reply:
We can do this when API will be finalized.